vortex-cli/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "vortex-cli"
+description = "a small but might tool for working with Vortex files"
+version.workspace = true
+homepage.workspace = true
+repository.workspace = true
+authors.workspace = true
+license.workspace = true
+keywords.workspace = true
+include.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+readme.workspace = true
+categories.workspace = true
+
+[dependencies]
+bytes = { workspace = true }
+clap = { version = "4", features = ["derive"] }
+crossterm = "0.28"
```

Comment:
Should we move these into root Cargo.toml and use `{workspace = true}`?

Reply:
Yea I was on the fence about this one. The "rusty" thing to do for a binary crate inside of a workspace is seems to be put all deps that are specific to that bin in the local Cargo.toml

E.g. datafusion-cli: https://github.com/apache/datafusion/blob/main/datafusion-cli/Cargo.toml

uv seems to use workspace deps though: https://github.com/astral-sh/uv/blob/main/crates/uv-cli/Cargo.toml

Can move things to workspace